.github/workflows/iam-integrations.yaml

```diff
@@ -12,43 +12,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  ldap-test:
-    name: LDAP Tests with Go ${{ matrix.go-version }}
+  iam-matrix-test:
+    name: "[Go=${{ matrix.go-version }}|ldap=${{ matrix.ldap }}|etcd=${{ matrix.etcd }}|openid=${{ matrix.openid }}]"
     runs-on: ubuntu-latest
 
     services:
       openldap:
-        image: quay.io/minio/openldap
+        image: donatello/openldap
```

Comment:
anything changed that is needed for this - same commitID

```
4a5b029b01a3e0df4a982a67c19f4b437bc2bb1749cc1e55c769fd2026df2bb2
```

Reply:
The ldif file is updated for ldap image